### ISSUE TITLE
Issue Collection for Course Structure XSD

### ISSUE THREAD

**Issue by svrnm** [AICC/CMI-5_Spec_Current#113]
Hi,
there are some further improvments and bug fixes I'd like to add or fix:

- ```<au>``` can only occur once within a ```<block>```. The specification of ```auType``` (and blockType) is missing the attribute maxOccurs="unbounded".
- The title and description of an objective are called objectiveTitle and objectiveDescription. That is not consistent with course, block or au. So I'd like to call them title and description, too.
- I'd like to enforce uniqueness for the id of an au.
- The element ```<langstring>``` is not specified within the XSD. I can add it, but actually I would prefer to follow the Best Practices for XML Internationalization: http://www.w3.org/TR/xml-i18n-bp/
- If authenticationMethod is "OAuth" I have  to specify this for each au. So it like to propose to put an optional default "authenticationMethod" on course level. The same might be true with "launchMethod".

**Comment by svrnm** [comment]
* The runtime reference states that the object of an xAPI statement is the AU and that this object has an activity type like "assessment", "tutorial", ..., "video". As far as I understand this definition this type is fixed, so it might be useful to an LMS to have this information already within the course structure:

```xml
<au activityType="video" ...>
...
</au>
<au activityType="assessment" ...>
```

This makes it possible to display the type to the user before launching the course and to use this information for evaluation and reporting. 
* ```launchParameters``` and ```entitlementKey```  should not be required attributes, since they might be empty.

**Comment by svrnm** [comment]
Most of these issues are resolved with my pull request #119.
